src/measure_theory/constructions/borel_space.lean

```diff
@@ -817,12 +817,12 @@ lemma ae_measurable_restrict_of_monotone_on [linear_order β] [order_closed_topo
 have this : monotone (f ∘ coe : s → α), from λ ⟨x, hx⟩ ⟨y, hy⟩ (hxy : x ≤ y), hf hx hy hxy,
 ae_measurable_restrict_of_measurable_subtype hs this.measurable
 
-lemma measurable_of_antimono [linear_order β] [order_closed_topology β] {f : β → α}
+lemma antitone.measurable [linear_order β] [order_closed_topology β] {f : β → α}
```

Comment:
Why did you change this to dot notation but not the monotone one?

Reply:
The monotone one had an alias, but it is indeed quite useless as `measurable_of_monotone` is strictly worse than `monotone.measurable`.